src/fillPullBody.js

```diff
@@ -4,11 +4,13 @@ chrome.runtime.onMessage.addListener(function(request){
   const prBodyElement = document.getElementById('pull_request_body');
   if(request.fillPR && prBodyElement){
     chrome.storage.sync.get('prTemplate', function({ prTemplate }){
-      if(prBodyElement && !prBodyElement.value.includes(prTemplate)){
+      if(!prBodyElement.value.includes(prTemplate)){
         if(prBodyElement.value){
           prBodyElement.value += '\n';
         }
         prBodyElement.value = prBodyElement.value + prTemplate;
+      } else if(prBodyElement.value.includes(prTemplate)){
```

Comment:
shouldn't need the `if` section here, as the only way the `else` would happen is if it does include the value, right?